lib/orator/index.test.js

```diff
@@ -332,6 +332,28 @@ test('hasPermission/dm command in guild', async (t) => {
   t.false(ok)
 })
 
+test('_tryToExecute/no response', async (t) => {
```

Comment:
try to execute doesn't care about the command response, lets run a test on process command response to make sure it doesn't do anything when the command response is undefined